### ISSUE TITLE
[Error] Failed to run the application in iOS

### ISSUE THREAD

**Issue by levani** [Microsoft/vscode-react-native#712]
## Actual Behavior

When running the following command `React Native: Run iOS on Device` I'm getting this error:

> [Error] Failed to run the application in iOS
> 
> Stack: TypeError: Failed to run the application in iOS: Cannot read property 'message' of undefined
> 	at Function.getWrappedError (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/src/common/error/internalError.js:35:16)
> 	at Function.wrapError (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/src/common/error/errorHelper.js:18:44)
> 	at resultOfCode.done.reason (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/src/common/entryPointHandler.js:40:72)
> 	at _rejected (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:844:24)
> 	at /Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:870:30
> 	at Promise.when (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:1122:31)
> 	at Promise.promise.promiseDispatch (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:788:41)
> 	at /Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:604:44
> 	at runSingle (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:137:13)
> 	at flush (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/node_modules/q/q.js:125:13)
> 	at _combinedTickCallback (internal/process/next_tick.js:73:7)
> 	at process._tickCallback (internal/process/next_tick.js:104:9)

## Expected Behavior

Run application on device

## Software versions
- React-Native VS Code extension version: 0.6.10
- VSCode version: 1.23.1
- OS platform and version: macOS 10.13.4
- NodeJS version: 8.11.2
- React Native version: 0.55.4
- Expo SDK version (include if relevant):

## Outputs (Include if relevant)
- Output of the Debug Console (View -> Toggle Debug Console):
```
PASTE OUTPUT HERE
```
- Output of the React-Native output channel (View -> Toggle Output -> Select React-Native in ListBox):
```
PASTE OUTPUT HERE
```
- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab):
```

console.ts:136 [Extension Host] vscode-icons is active!
console.ts:136 [Extension Host] rejected promise not handled within 1 second
t.log @ console.ts:136
t._logExtensionHostMessage @ extensionHost.ts:393
(anonymous) @ extensionHost.ts:210
emitTwo @ events.js:106
emit @ events.js:194
process.nextTick @ internal/child_process.js:766
_combinedTickCallback @ internal/process/next_tick.js:73
_tickCallback @ internal/process/next_tick.js:104
mainThreadExtensionService.ts:43 [vsmobile.vscode-react-native]Error while executing command 'react-native -v'
e.$onExtensionRuntimeError @ mainThreadExtensionService.ts:43
e._doInvokeHandler @ rpcProtocol.ts:265
e._invokeHandler @ rpcProtocol.ts:250
e._receiveRequest @ rpcProtocol.ts:196
e._receiveOneMessage @ rpcProtocol.ts:178
(anonymous) @ rpcProtocol.ts:111
(anonymous) @ rpcProtocol.ts:307
e.fire @ event.ts:140
(anonymous) @ ipc.net.ts:81
emitOne @ events.js:96
emit @ events.js:191
readableAddChunk @ _stream_readable.js:178
Readable.push @ _stream_readable.js:136
onread @ net.js:560
mainThreadExtensionService.ts:44 Error: Error while executing command 'react-native -v'
	at Function.getInternalError (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/src/common/error/errorHelper.js:11:16)
	at ChildProcess.spawnedProcess.once (/Users/lm/.vscode/extensions/vsmobile.vscode-react-native-0.6.10/src/common/node/childProcess.js:44:58)
	at Object.onceWrapper (events.js:293:19)
	at emitTwo (events.js:106:13)
	at ChildProcess.emit (events.js:194:7)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
e.$onExtensionRuntimeError @ mainThreadExtensionService.ts:44
e._doInvokeHandler @ rpcProtocol.ts:265
e._invokeHandler @ rpcProtocol.ts:250
e._receiveRequest @ rpcProtocol.ts:196
e._receiveOneMessage @ rpcProtocol.ts:178
(anonymous) @ rpcProtocol.ts:111
(anonymous) @ rpcProtocol.ts:307
e.fire @ event.ts:140
(anonymous) @ ipc.net.ts:81
emitOne @ events.js:96
emit @ events.js:191
readableAddChunk @ _stream_readable.js:178
Readable.push @ _stream_readable.js:136
onread @ net.js:560
console.ts:136 [Extension Host] found 20 files with exports
console.ts:136 [Extension Host]% Object {}
```

screenshot just in case:

![screenshot at jun 03 15-23-33](https://user-images.githubusercontent.com/184472/40886045-268c0a32-6742-11e8-87da-d52ac1a5998c.png)

**Comment by ruslan-bikkinin** [comment]
Hi @levani and thanks for you report. 
[This PR](https://github.com/Microsoft/vscode-react-native/pull/711) that already merged to master should fix this error. Could you please try it on your machine? To do that please follow these steps:
1. Run these commands:
```
git clone git@github.com:Microsoft/vscode-react-native.git
cd vscode-react-native
npm i 
```
2. Open `vscode-react-native` directory in VS Code, click on `Debug` icon, then select `Quick Launch Extension` in drop down menu and run it.
3. Wait for another instance of VS Code to be appeared (it will be named like `Extension Development Host...`), then open your project in it and try to debug it regular way.

Please let us know whether it resolves your issue.

**Comment by ruslan-bikkinin** [comment]
I am closing this due to fix already included in new version v0.6.11. Please, feel free to reopen if needed.
